controllers/api/commentRoutes.js

```diff
@@ -1,32 +1,52 @@
 const router = require('express').Router();
-const { Comment } = require('../../models');  
+const { User, Book, Comment } = require('../../models');  
 const withAuth = require('../utils/auth');
```

Comment:
Just like you did when requiring 'models', you'll need to back out twice (add another '../') to access the 'withAuth' middleware function.